src/infrastructure/webhook/webhook-service.test.ts

```diff
@@ -74,6 +74,7 @@ describe('WebhookService', () => {
 		jest
 			.spyOn(figmaTeamRepository, 'getByWebhookId')
 			.mockResolvedValue(figmaTeam);
+		jest.spyOn(figmaTeamRepository, 'updateAuthStatus').mockResolvedValue();
```

Comment:
Realised I forgot to mock this call, not sure how the test was passing before 🤔

Reply:
Haha, nice catch! You, likely, had your test database running, so this unit test wrote to the database.